app/Kconfig

```diff
@@ -331,6 +339,7 @@ menu "KSCAN Settings"
 config ZMK_KSCAN_EVENT_QUEUE_SIZE
 	int "Size of the event queue for KSCAN events to buffer events"
 	default 4
+  default 64 if ZMK_PLOVER_HID
```

Comment:
Do we really need it this high? Can't imagine anyone w/ this many fingers... even if they are a steno nut.

Reply:
The largest chord in an extended steno system is 29 keys at the same time (assuming a split S-key and four star keys) + a number bar stroke. In the standard english system there are two fewer keys in the max chord. So I just doubled that amount (to account for both presses and releases) and rounded it up to a power of two. But I can of course try to lower that, but that might need some testing.